### PR TITLE
Test cli

### PR DESCRIPTION
**Pull request by martinjzhang**
1. Printing more info for ./bin/scdrs; added `sc.pp.pca` before `sc.pp.neighbors` (otherwise sc.pp.neighbors will use all PCs instead of the number specified by `n_pcs `; we didn't find this before because we used all PCs for the toy data)
2. Similarly, added `sc.pp.pca` for `compute_downstream.py` 
3. Updated files in ./scdrs/data/res to those generated using `scdrs` CLI (I checked that they are consistent)
4. Added following files used by unit test: 
    - scdrs/data/toydata_gs_mouse.ref_Ctrl20_CovConstCovariate.full_score.gz 
    - scdrs/data/toydata_gs_mouse.ref_Ctrl20_CovConstCovariate.scdrs_cell_corr 
    - scdrs/data/toydata_gs_mouse.ref_Ctrl20_CovConstCovariate.scdrs_gene
    - scdrs/data/toydata_gs_mouse.ref_Ctrl20_CovConstCovariate.scdrs_group.cell_type
    - scdrs/data/toydata_gs_mouse.ref_Ctrl20_CovConstCovariate.score.gz
5. Minor changes to scdrs/method.py
6. Added unit tests in tests/test_method_downstream.py (could you review `test_downstream_group_analysis `)?
7.

### CODE REVIEW

File: compute_downstream.py [hunk 1]
```diff
@@ -151,7 +151,8 @@ def main(args):
 
     # Compute connectivities if need to do cell type-level analysis
     if (len(CELLTYPE_LIST) > 0) & ("connectivities" not in adata.obsp):
-        sc.pp.neighbors(adata, n_neighbors=10, n_pcs=40)
+        sc.pp.pca(adata, n_comps=20)
```
Comment (KangchengHou) [inline compute_downstream.py:154]: this is just to note that these are set as constants. (I guess for testing) and they will get removed anyway.

**Comment by KangchengHou** [comment]
I don't follow on (1): if i call `sc.pp.neighbors(adata, n_pcs=10)` for an adata without precomputed PCs. Is this saying it will ignore the n_pcs parameter?

**Review by KangchengHou — APPROVED** [review]

**Comment by martinjzhang** [comment]
> I don't follow on (1): if i call `sc.pp.neighbors(adata, n_pcs=10)` for an adata without precomputed PCs. Is this saying it will ignore the n_pcs parameter?

In this case, I think it will call `sc.pp.pca` with its default parameters  `n_comps =min(50, min(adata.shape)-1)`, and then use all the computed PCs for `sc.pp.neighbors(adata)`. I checked it in the script but I will try to create a minimal example to illustrate this.

File: compute_downstream.py (hunk 1 above)
Comment (martinjzhang) [reply compute_downstream.py:154]: Yes I wanted to make minimal change to old scripts while still making it work.

**Pull request merged**